src/Trakx.Coinbase.Custody.ApiClient/CoinbaseCustodyApiConfiguration.cs

```diff
@@ -1,14 +1,27 @@
 ﻿using System.ComponentModel.DataAnnotations;
+using Trakx.Utils.Attributes;
 
 namespace Trakx.Coinbase.Custody.ApiClient
 {
     public class CoinbaseCustodyApiConfiguration
     {
 #nullable disable
-        [Required] public string BaseUrl { get; set; }
-        [Required] public string AccessKey { get; set; }
-        [Required] public string PassPhrase { get; set; }
+        [Required]
+        [SecretEnvironmentVariable("CoinbaseCustodyApiConfiguration__BaseUrl")]
+        public string BaseUrl { get; set; }
+
+        [Required]
+        [SecretEnvironmentVariable("CoinbaseCustodyApiConfiguration__AccessKey")]
+        public string AccessKey { get; set; }
+
+        [Required]
+        [SecretEnvironmentVariable("CoinbaseCustodyApiConfiguration__PassPhrase")] 
+        public string PassPhrase { get; set; }
+
+        [SecretEnvironmentVariable("CoinbaseCustodyApiConfiguration__InitialRetryDelayInMilliseconds")]
```

Comment:
the ones which are not secret should simply be shared in the appsettings when you commit
this is just to indicate what we consider secret, and to generate readme doc automatically for other devs to get the .env file ready rapidly.
also cf other PR comments, I think 0.1.11 lets you not add the variable name